datalad_ria/conftest.py

```diff
@@ -5,7 +5,7 @@
 ]
 
 from .tests.fixtures import (
-    ria_sshserver,
```

Comment:
was this an unintended deletion? My tests fail locally if I test the PR code, but succeed if I bring the `ria_sshserver` import back

Reply:
woops, yes, that must have been a mishap during conflict resolution! Thanks for catching this!